### PR TITLE
chore: Update workflows

### PR DESCRIPTION
**Pull request by susnux**
* Drop `command-rebase` this is now native on Github
* Update `lint-eslint` from templates
* Update `lint-stylelint` from templates
* Replace `npm-test` with `node-test` from the templates
* Drop `node` because the build is already included in `node-test` so that will fail if build is not possible
* Update `npm-publish` 
  * Also adjust to automatically set correct tag for beta and alpha releases

### CODE REVIEW

**Review by mejo- — APPROVED** [review]

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -11,39 +11,51 @@ on:
...
   contents: read
+  packages: write
 
 jobs:
   publish:
     runs-on: ubuntu-latest
 
     name: Build and publish to npm
+    env:
+      RELEAS_GROUP: ${{ (contains(github.ref, 'beta') || contains(github.ref, 'alpha')) && 'next' || 'latest' }}
```
Comment (Antreesy) [inline .github/workflows/npm-publish.yml:22]: Typo?

**Comment by mejo-** [comment]
Also it seems like the branch merge requirements need to be adjusted. It's waiting for required job "Node" which gets removed with this PR.

**Review by skjnldsv — APPROVED** [review]

**Comment by susnux** [comment]
> branch merge requirements need to be adjusted#

Yes but waiting for this to be approved :wink:

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (susnux) [reply .github/workflows/npm-publish.yml]: Good catch!

**Pull request merged**